src/components/modals/EntityExtractor.tsx

```diff
@@ -3,23 +3,23 @@
  * Licensed under the MIT License.
  */
 import * as React from 'react'
+import * as Util from '../../Utils/util'
+import * as CLM from '@conversationlearner/models'
+import * as OF from 'office-ui-fabric-react'
+import * as ToolTips from '../ToolTips/ToolTips'
+import * as ExtractorResponseEditor from '../ExtractorResponseEditor'
```

Comment:
I noticed this in other reviews. Is this just because you like wildcard imports at top?

I think there are some extensions or Prettier has import sorting options that way you don't have to do this manually if you are.